src/components/chatbox/chatbox.tsx

```diff
@@ -0,0 +1,551 @@
+import React, { useEffect, useState, useRef } from 'react'
+
+// https://helios.hashicorp.design/icons/library
+import { IconArrowDownCircle16 } from '@hashicorp/flight-icons/svg-react/arrow-down-circle-16'
+import { IconBulb16 } from '@hashicorp/flight-icons/svg-react/bulb-16'
+import { IconDiscussionCircle16 } from '@hashicorp/flight-icons/svg-react/discussion-circle-16'
+import { IconLoading24 } from '@hashicorp/flight-icons/svg-react/loading-24'
+import { IconSend24 } from '@hashicorp/flight-icons/svg-react/send-24'
+import { IconStopCircle24 } from '@hashicorp/flight-icons/svg-react/stop-circle-24'
+import { IconUser16 } from '@hashicorp/flight-icons/svg-react/user-16'
+import { IconWand24 } from '@hashicorp/flight-icons/svg-react/wand-24'
+import ms from 'ms'
+import { z } from 'zod'
+
+import { useMutation } from '@tanstack/react-query'
+
+import useAuthentication from 'hooks/use-authentication'
+import IconTile from 'components/icon-tile'
+import Button from 'components/button'
+import Heading from 'components/heading'
+import Text from 'components/text'
+
+import cn from 'classnames'
```

Comment:
⛏️ another small nitpick, could you organize the imports ?

Reply:
Also can you use `classNames` just for consistency sake with the rest of the repo?

Reply:
updated in 47cfb1c13